README.md

```diff
@@ -941,3 +943,26 @@ Onyx derived values are special Onyx keys which contain values derived from othe
 3. **Document derived values**
    - Explain the purpose and dependencies
    - Document any special cases or performance considerations
+
+# canBeMissing onyx param
+
+Context https://expensify.slack.com/archives/C03TQ48KC/p1741208342513379
```

Comment:
This is an internal Slack link so the majority of App contributors can't access it, is there a thread in #expensify-open-source we could link to instead? Or just skip this line?

Reply:
I don't think there is, but let's keep this anyway? At least we will be able to see the context...